containers/kubernetes/reference-content/exposing-services.mdx

```diff
@@ -7,21 +7,21 @@ content:
   paragraph: Exposing Kubernetes services to the internet
 tags: kubernetes k8s wildcard clusterip nodeport node-port loadbalancer load-balancer ingress ingress-controller
 dates:
-  validation: 2023-10-30
+  validation: 2023-12-29
   posted: 2023-10-25
 categories:
   - kubernetes
 ---
 
 By default, Kubernetes clusters are not exposed to the internet. This prevents external users from being able to access the application deployed in your cluster.
 
-There are a number of different ways to expose your cluster to the internet. In this documentation, we explain and compare some of the different ways to do so, and provide information to help you get started.
+There are a number of different ways to expose your cluster to the internet. In this documentation, we explain and compare different ways to do so, and provide information to help you get started.
 
 ## Comparison of cluster exposure methods
 
-In Kubernetes, you generally need to use a [Service](/containers/kubernetes/concepts/#services) to expose an application in your cluster to the internet. A service groups together pods performing the same function (e.g. running the same application) and defines how to access them. 
+In Kubernetes, you generally need to use a [Service](/containers/kubernetes/concepts/#services) to expose an application in your cluster to the internet. A service groups together pods performing the same function (e.g. running the same application) and defines how to access them.
 
-The most basic type of service is **ClusterIP**, but this only provides internal access, from within the cluster, to the defined pods. The **NodePort** and **LoadBalancer** services both provide external access. **Ingress** (which is not a service but an API object inside a cluster) combined with an explicitly-created **Ingress Controller** is another way to expose the cluster.
+The most basic type of service is **clusterIP**, but this only provides internal access, from within the cluster, to the defined pods. The **NodePort** and **LoadBalancer** services both provide external access. **Ingress** (which is not a service but an API object inside a cluster) combined with an explicitly created **ingress controller** is another way to expose the cluster.
```

Comment:
I would leave the hyphen in explicitly-created since it is acting as an adjective before a noun?